packages/cli/src/lib/transform.ts

```
@@ -2,33 +1,61 @@
+import { execFileSync, spawnSync } from "child_process";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)